### PR TITLE
Fix for responsiveness and basic accessibility of file list page, while adding cross-env.

### LINKED ISSUE #30: 🐛 [BUG] - Bad responsiveness on file's tab.
### Description

The files tab has a very squished look on a phone. Some margins and separations might be added to keep it clean.
Here's what it currently looks like.
image

### Reproduction URL

https://storz.pages.dev/app/myFiles

### Reproduction steps

```bash
1. Go to 'https://storz.pages.dev/app/myFiles'
2. Switch to a smaller screen in devtools
4. See the issue
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Logs

_No response_

### Browsers

Firefox, Chrome, Safari, Microsoft Edge

### OS

Windows, Linux

### PR DESCRIPTION
**Pull request by Lofty-Brambles**
# Description

`1.` Fixes the squished text and minimalizes the text on screen on mobile devices.
`2.` Fixed accessibility of the page's images.
`3.` Adds `cross-env` for any and all env-vars on windows. [E.g.: The `build` command for client.]

Fixes #30 (issue)

## Type of change
* Styling
* OS support

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Lofty-Brambles** [comment]
Done 👍

File: client/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "@tippyjs/react": "^4.2.6",
     "aos": "^2.3.4",
     "axios": "^0.27.2",
+    "cross-env": "^7.0.3",
```
Comment (anomic30) [inline client/package.json:12]: As of now, revert to previous changes i.e "dotenv"

File: client/package.json [hunk 2]
```diff
@@ -29,7 +30,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "GENERATE_SOURCEMAP=false react-scripts build",
+    "build": "cross-env GENERATE_SOURCEMAP=false react-scripts build",
```
Comment (anomic30) [inline client/package.json:33]: Remove cross-env as of now.

File: client/src/pages/myFiles/MyFiles.css [hunk 3]
```diff
@@ -63,6 +70,10 @@
     width: 70%;
 }
 
+#size-col {
+	text-align: center;
```
Comment (anomic30) [inline client/src/pages/myFiles/MyFiles.css:74]: Make it left aligned

**Review by anomic30 — CHANGES_REQUESTED** [review]
Please go through the review.

File: client/src/pages/myFiles/MyFiles.css (hunk 3 above)
Comment (Lofty-Brambles) [reply client/src/pages/myFiles/MyFiles.css]: Should be done

**Review by anomic30 — CHANGES_REQUESTED** [review]
Add the "dotenv" package

**Review by anomic30 — APPROVED** [review]
LGTM!

**Pull request merged**